includes/classes/bmz_image_handler.class.php

```diff
@@ -92,6 +93,20 @@ public function __construct($src, $width, $height)
             $this->debugLogFile = DIR_FS_LOGS . "/ih_debug-$logfile_suffix.log";
         }
 
+        // -----
+        // PHP provides resizing support for .jpg/.jpeg, .gif and .png files.  If the submitted
+        // image's extension isn't one of those supported, there's no sense in checking any of the
+        // file-related attributes!
+        //
+        // If that case if found, set a processing flag (used by the 'handle_image' function) to indicate
+        // as such and perform a quick return.
+        //
+        $this->file_is_supported = true;
```

Comment:
I would have coded this
```
       $this->file_is_supported = h_image_supported($src);
        if ($this->file_is_supported === false) {
            return;
        }
```

Reply:
I agree; update forthcoming.  That was left-over from where I was originally heading down a different implementation path, thanks for the catch!

Reply:
no probs glad to help.

Reply:
That update just pushed addresses this change request.